packages/sanity/src/core/form/studio/uploads/accepts.ts

```diff
@@ -28,7 +28,14 @@ export function accepts(file: FileLike, acceptedFiles: string | string[]): boole
     const validType = type.trim().toLowerCase()
 
     if (validType.charAt(0) === '.') {
-      return fileName.toLowerCase().endsWith(validType)
+      if (fileName) {
+        return fileName.toLowerCase().endsWith(validType)
+      }
+      // If we do not have a valid fileName and validType is an extension, we
```

Comment:
Looks good to me!

nit: I had to take a little time to understand what was going on here. If I understand correctly, we pass true here to not get an error state since the filename is not available. However, when the file is dropped, the `accept` rule will be enforced later on and an invalid type will not be accepted?

Reply:
That is correct!